### PR TITLE
Make attribute conditionals more consistent

### PR DESCRIPTION
**Pull request by ekohl**
This uses the pattern that if there's an ifdef::X[] for one sentence and then another for the others, it uses ifndef::X[] for the other bits.  Sort of like an if/else condition. This makes it easier to see what's related and you don't miss a sentence on another build.

This should have no content change: the diff is expected to be empty.

I'm picking 3.5 since I worry about complex cherry picks. On the other hand, it may also complicate future cherry picks. Up for debate what we prefer.

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.5/Katello 4.7
* [ ] Foreman 3.4/Katello 4.6
* [ ] Foreman 3.3/Katello 4.5
* [ ] Foreman 3.2/Katello 4.4
* [ ] Foreman 3.1/Katello 4.3
* For Foreman 3.0 or older, please create a separate PR.
* We do not accept PRs for Foreman 2.3 or older.

### CODE REVIEW

**Comment by ekohl** [comment]
@maximiliankolb I think the diff is actually correct and it was wrong before. Mind having a look?

File: guides/common/modules/proc_configuring-satellite-for-outgoing-emails.adoc [hunk 1]
```diff
@@ -58,7 +58,7 @@ The `SMTP username` and `SMTP password` specify the login credentials for the SM
 +
 For security reasons, both Sendmail location and Sendmail argument settings are read-only and can be only set in `/etc/foreman/settings.yaml`.
 Both settings currently cannot be set via `{foreman-installer}`.
-ifdef::foreman,katello,orcharhino[]
+ifndef::satellite[]
```
Comment (maximiliankolb) [inline guides/common/modules/proc_configuring-satellite-for-outgoing-emails.adoc:61]:
```suggestion
ifndef::satellite,orcharhino[]
```

**Review by maximiliankolb — COMMENTED** [review]
One small request, rest LGTM. Esp. the ifdefs for images are easier to read. Thanks!

**Comment by ekohl** [comment]
159601dfc..6447fcec4  3.5 -> 3.5

**Pull request merged**